### ISSUE TITLE
Installing error: hdiutil: attach failed - image not recognised

### ISSUE THREAD

**Issue by a-million-starflakes** [75lb/handbrake-js#46]
"react-native": "0.59.9",

npm install handbrake-js

```
 handbrake-js@4.0.0 postinstall /.../node_modules/handbrake-js
> node scripts/install.js

fetching: http://download.handbrake.fr/releases/1.2.2/HandBrakeCLI-1.2.2.dmg
extracting: HandbrakeCLI
/.../node_modules/handbrake-js/scripts/install.js:62
      if (err) throw err
               ^

Error: Command failed: hdiutil attach mac.dmg
hdiutil: attach failed - image not recognised
```

Any ideas?

**Comment by 75lb** [comment]
well this link returns a 403: https://download.handbrake.fr/releases/1.2.2/HandBrakeCLI-1.2.2.dmg

**Comment by 75lb** [comment]
i'll look into it

**Comment by 75lb** [comment]
Looks like handbrake.fr are having issues with their download servers - sometimes the download works, sometimes it doesn't (403). 

I've updated handbrake-js to download the HandbrakeCLI binary from github instead of handbrake.fr. Fixed in handbrake-js@4.0.1, cheers.
